### ISSUE TITLE
Fuzzy language matching.

### ISSUE THREAD

**Issue by Lattyware** [Lattyware/massivedecks#113]
As notes in #111, our current strategy for matching languages (exact string match) isn't ideal.

We really want to do some standardising & fuzzy matching (e.g: https://github.com/LuminosoInsight/langcodes)

**Comment by TheChilliPL** [comment]
A slightly similar issue is with Many Decks—e.g. filtering language pl (Polish) gives different decks than pl-PL (Polish—Poland)

**Comment by Lattyware** [comment]
Yeah, that issue is being tracked at manydecks#11—the real issue here is that `pl-PL` is redundant and should be simplified to just `pl`, but it is hard to explain that to users doing it manually, and I haven't found a good data source for automating or suggesting that (it is non-trivial, as what is redundant is entirely dependent on people's expectations, even if you did the "obvious" solution of tracking the home country and marking it as redundant, that falls over in some cases—e.g: `en-GB` which is generally considered distinct from `en`, and of course if that distinction matters comes down to the deck. I'm English, and if I'm writing a deck I'd normally just mark it as `en`, but if it is using British slang I might want to use `en-GB`, etc...)
